algorithm: "TD3"

training_steps: 50000

max_speed: 4.0
max_steer: 0.4
number_of_beams: 20
range_finder_scale: 10
n_scans: 2

reward: 'TAL'

reward_tal_constant: 0.2
reward_tal_inv_scales: [0.8, 4]
tal_racetrack_set: "mu90"

cth_max_speed: 3.0
cth_distance_weight: 1
cth_speed_weight: 1

progress_weight: 100

tal_constant_lookahead: 0.8
tal_variable_lookahead: 0.6
constant_speed: 3
